soot-infoflow-android/src/soot/jimple/infoflow/android/entryPointCreators/AndroidEntryPointConstants.java

```diff
@@ -135,7 +135,8 @@
 
 	private static final String[] fragmentMethods = { FRAGMENT_ONCREATE, FRAGMENT_ONDESTROY, FRAGMENT_ONPAUSE,
 			FRAGMENT_ONATTACH, FRAGMENT_ONDESTROYVIEW, FRAGMENT_ONRESUME, FRAGMENT_ONSTART, FRAGMENT_ONSTOP,
-			FRAGMENT_ONCREATEVIEW, FRAGMENT_ONACTIVITYCREATED, FRAGMENT_ONVIEWSTATERESTORED, FRAGMENT_ONDETACH };
+			FRAGMENT_ONCREATEVIEW, FRAGMENT_ONACTIVITYCREATED, FRAGMENT_ONVIEWSTATERESTORED, FRAGMENT_ONDETACH,
+			FRAGMENT_ONSAVEINSTANCESTATE, FRAGMENT_ONVIEWCREATED };
```

Comment:
Hier bin ich mir nicht sicher, ob wir das generisch abhandeln müssen, oder ob wir auch hier explizit die Views in den Fragments analysieren und erzeugen, d.h. eine Referenz auf die korrekte View übergeben können. Vielleicht kann dazu auch @MarcMil etwas sagen.

Reply:
Auch hier werden im `entryPointCreator` alle Konstanten einzeln verwendet, um die dummyMain zu erzeugen. Die `fragmentMethodList` wird wieder (mindestens hauptsächlich) verwendet, um zu prüfen, ob eine Methode eine Lifecycle-Methode ist.